gatsby-config.js

```diff
@@ -1,3 +1,7 @@
+const constants = require(`./src/constants`)
+
+console.log(`env: ${constants.environment}`)
```

Comment:
Nix?

Reply:
Going to leave this one here for now. Just adds a log message during the build process. :)